rh_ui/templates/info_pages/cookies.html

```diff
@@ -106,26 +106,13 @@ <h3>{{ _("Cookies that measure website use") }}</h3>
                         {
                             "tds": [
                                 {
-                                    "value": "<code>_gid</code>"
+                                    "value": "<code>_ga_<container-id></code>"
```

Comment:
When viewing the page this is only rendering as `_ga_`, I think the angle braces probably need to be escaped.